supertokens_python/recipe/jwt/utils.py

```diff
@@ -51,3 +52,17 @@ def validate_and_normalise_user_input(
         jwt_validity_seconds = 3153600000
 
     return JWTConfig(override, jwt_validity_seconds)
+
+
+async def get_issuer_domain_or_throw_error(
+    issuer: Optional[str], app_info: AppInfo, user_context: Any
+) -> str:
+    if issuer is None:
+        if app_info.initial_api_domain_type == "string":
+            api_domain_res = await app_info.api_domain(None, user_context)
+            api_domain = api_domain_res.get_as_string_dangerous()
+            return api_domain
+        raise Exception(
+            "Please pass issuer as a string to the function or pass api_domain as string in supertokens.init"
+        )
```

Comment:
get the request object from the user context and if it exists, call the function, else throw an error